backend/src/mirrors_qa_backend/schemas.py

```diff
@@ -57,20 +61,23 @@ class Paginator(BaseModel):
     last_page: int | None = None
 
 
-ISOCountryCode = Annotated[str, Field(min_length=2, max_length=2)]
+class Region(BaseModel):
+    code: ISO3166AlphaCode
```

Comment:
AFAIK those region (continent actually) codes are not standardized by ISO. It's small enough that the convention is used everywhere though.